src/PaymentMethods/OneClickPaymentMethod.php

```diff
@@ -85,7 +85,7 @@ public function getTemplate(): ?string
      */
     public function getLogo(): string
     {
-        return 'https://checkoutshopper-live.adyen.com/checkoutshopper/images/logos/medium/card.png';
+        return 'card.png';
```

Comment:
Could you pls create a follow up ticket to download all the icons for all the card types available for the merchant like: visa, bcmc etc?
For now it's a perfect solution but since for the one click payment methods we know the card type we can also show the logo for that.